jack/_posts/2017-10-16-day-33.md

```diff
@@ -14,3 +14,4 @@ Things to write about:
 
 ***
 
+Today in Social Studies I wrote down if I could live up to the rules the Puritans had for children, and why. Then I wrote which rules I thought are relevant today. After that I wrote a list of manners kids should follow today. I also discussed about the Salem Witch Trials.
```

Comment:
Better sentences:

"I wrote down if" = "I wrote about whether"

"I also discussed about the Salem Witch Trials" = "I discussed the Salem Witch Trials with a parent"